src/Nordea/Components/Button.elm

```diff
@@ -240,6 +259,37 @@ variantStyle variant =
                 , hover [ backgroundColor Colors.grayHover, arrowIconMovement (rem 1) ]
                 ]
 
+        SmallCard ->
+            let
+                arrowIconMovement movementLengthRight =
+                    descendants
+                        [ everything
+                            [ withClass "arrowicon"
+                                [ transforms [ translateY (pct -50), translateX movementLengthRight ]
+                                , transition [ Css.Transitions.transform3 150 0 easeOut ]
+                                , position absolute
+                                , top (pct 50)
+                                , right (rem 1.5)
+                                ]
+                            ]
+                        ]
+            in
+            batch
+                [ displayFlex
```

Comment:
Is it better to pull out the common styling of Card and SmallCard into a `cardStyle` variable?